diffs/AaveV3Ethereum_OnboardingETHxToAaveV3Ethereum_20231212_before_AaveV3Ethereum_OnboardingETHxToAaveV3Ethereum_20231212_after.md

```diff
@@ -0,0 +1,123 @@
+## Reserve changes
+
+### Reserves added
+
+#### ETHx ([0xA35b1B31Ce002FBF2058D22F30f95D405200A15b](https://etherscan.io/address/0xA35b1B31Ce002FBF2058D22F30f95D405200A15b))
+
+| description | value |
+| --- | --- |
+| decimals | 18 |
+| isActive | true |
+| isFrozen | false |
+| supplyCap | 10,000 ETHx |
+| borrowCap | 1,000 ETHx |
+| debtCeiling | 0 $ |
+| isSiloed | false |
+| isFlashloanable | true |
+| eModeCategory | 0 |
+| oracle | [0xED65C5085a18Fa160Af0313E60dcc7905E944Dc7](https://etherscan.io/address/0xED65C5085a18Fa160Af0313E60dcc7905E944Dc7) |
```

Comment:
@rustboyar do you have time to deploy a new oracle aligned with this but exposing decimals?